.github/workflows/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 🚀 Feature request
+about: Suggest a new feature for this project
+labels: 'enhancement'
+---
+
+Features that impact the whole project (e.g. Adding a new OpenJDK distribution)are made over at the [TSC](https://github.com/AdoptOpenJDK/TSC/).
```

Comment:
Please change "distribution)are" to "distribution) are".

Reply:
Also, please change over at the [TSC] to either
- over at the [TSC repository]
or
- over at [TSC]

Reply:
Much better now, ta. :)